index.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const { Client, Collection, Intents } = require('discord.js');
+const { token } = require('./config.json');
+const Keyv = require('keyv');
+
+const userPrefs = new Keyv('sqlite://users.db');
+userPrefs.on('error', err => console.error('Keyv connection error:', err));
+
+const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.DIRECT_MESSAGES] });
+
+
+client.commands = new Collection();
+const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
+
+for (const file of commandFiles) {
+    const command = require(`./commands/${file}`);
+    client.commands.set(command.data.name, command);
+}
+
+client.once('ready', () => {
+    console.log('Ready!');
+});
+
+client.on('interactionCreate', async interaction => {
+    if (!interaction.isCommand()) {
+
+        client.commands.forEach((cmd => {
+            cmd.executeInteraction?.(interaction, userPrefs);
+        }))
+        return;
+    }
+
+    const command = client.commands.get(interaction.commandName);
+
+    if (!command) return;
+
+    try {
+        await command.execute(interaction, userPrefs);
+    } catch (error) {
+        console.error(error);
+        return interaction.reply({ content: 'There was an error while executing this command!', ephemeral: true });
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
        return interaction.reply({ content: "There was an error while executing this command!", ephemeral: true });
```